editor.planx.uk/src/@planx/components/Pay/Public/InviteToPayForm.tsx

```diff
@@ -0,0 +1,246 @@
+    };
+    const paymentRequest: PaymentRequest = await trigger(createPaymentRequest);
+    if (!error && paymentRequest.id)
+      redirectToConfirmationPage(paymentRequest.id);
+  };
+
+  // TODO: This fails in the editor
+  // TODO: Test on prod custom domains
+  const redirectToConfirmationPage = (paymentRequestId: string) => {
+    const params = new URLSearchParams({ paymentRequestId }).toString();
+    const inviteToPayURL = `./pay/invite?${params}`;
+    navigation.navigate(inviteToPayURL);
+  };
+
+  const formik = useFormik<FormValues>({
+    initialValues: {
+      payeeName: "",
+      payeeEmail: "",
+      applicantName: "",
+    },
+    onSubmit: (values) => onSubmit(values),
+    validateOnChange: false,
+    validateOnBlur: false,
+    validationSchema,
+  });
+
+  return isMutating ? (
+    <DelayedLoadingIndicator />
+  ) : (
+    <Card>
+      <Typography variant="h3">{title}</Typography>
+      <Typography variant="body2">
+        <ReactMarkdownOrHtml source={description} openLinksOnNewTab />
+      </Typography>
+      <StyledForm onSubmit={formik.handleSubmit}>
+        <InputLabel label="Full name (optional)" htmlFor="payeeName">
+          <Input
+            bordered
+            name="payeeName"
+            id="payeeName"
+            value={formik.values.payeeName}
+            onBlur={formik.handleBlur}
+            onChange={formik.handleChange}
+            errorMessage={
+              Boolean(formik.touched.payeeName && formik.errors.payeeName)
+                ? formik.errors.payeeName
+                : undefined
+            }
+            inputProps={{
+              "aria-describedby":
+                "Invite someone else to pay for this application - full name",
+            }}
+          />
+        </InputLabel>
+        <InputLabel label="Email" htmlFor="payeeEmail">
+          <Input
+            required
+            bordered
+            name="payeeEmail"
+            id="payeeEmail"
+            value={formik.values.payeeEmail}
+            onBlur={formik.handleBlur}
+            onChange={formik.handleChange}
+            errorMessage={
+              Boolean(formik.touched.payeeEmail && formik.errors.payeeEmail)
+                ? formik.errors.payeeEmail
+                : undefined
+            }
+            inputProps={{
+              "aria-describedby": [
+                "Invite someone else to pay for this application - email",
+                formik.touched.payeeEmail && formik.errors.payeeEmail
+                  ? formik.errors.payeeEmail
+                  : "",
+              ]
+                .filter(Boolean)
+                .join(" "),
+            }}
+          />
+        </InputLabel>
+        <Box>
+          <Typography variant="h3" pt={3} pb={2}>
+            Your details
+          </Typography>
+          <Typography variant="body2">
+            How should we refer to you in communications with your nominee?
+          </Typography>
+        </Box>
```

Comment:
based on feedback from UR/content team, I imagine we may want to at least move this subheader & description to be Editor configurable consistent with the other subheader/desc on this page - but very happy for this to be hardcoded now to start :+1:

Reply:
This is a very good shout - added to the editor form 👍